src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1040,7 +1040,7 @@ public function findTags()
      */
     public function findUnusedTags()
     {
-        return array_values(array_diff($this->findTags(), $this->usedTags));
+        return array_filter(array_values(array_diff($this->findTags(), $this->usedTags)));
```

Comment:
this would need to change the order of array_filter and array_values. otherwise array_values is useless.